pkg/agent/openflow/client_windows.go

```diff
@@ -23,10 +23,16 @@ import (
 	"fmt"
 	"net"
 
+	"antrea.io/antrea/pkg/agent/config"
 	"antrea.io/antrea/pkg/agent/openflow/cookie"
+	"antrea.io/antrea/pkg/agent/route"
 	binding "antrea.io/antrea/pkg/ovs/openflow"
 )
 
+// virtualSvcMAC is for gateway virtual IP to implement ClusterIP Service with AntreaProxy on Windows, in ARP
+// responder table.
+var virtualSvcMAC, _ = net.ParseMAC("aa:bb:cc:dd:ee:fe")
```

Comment:
I feel better to define virtual MAC together with virtual IP in the same file.

Reply:
Updated.